spec/pleaserun/configurable_spec.rb

```diff
@@ -113,7 +113,7 @@
     end
   end
 
-  context "with :default => ..." do
+  context "with default: ..." do
```

Comment:
this is a string. Odd you didn't update line 118 to use hash-colon, though? 

I still prefer hash rocket because I still support Ruby 1.8 for some projects. I see no reason to change at this time because of this.